src/modules/auth/styles/auth-page.sx.ts

```diff
@@ -0,0 +1,46 @@
+const authPageStyles = {
```

Comment:
Could you rename your style files to .styles.ts

Reply:
ok